python/tvm/autotvm/tuner/xgboost_cost_model.py

```diff
@@ -161,7 +162,8 @@ def _reset_pool(self, space, target, task):
         _extract_space = space
         _extract_target = target
         _extract_task = task
-        self.pool = multiprocessing.Pool(self.num_threads)
+        self.pool = popen_pool.PopenPoolExecutor(self.num_threads)
```

Comment:
`multiprocessing.pool()` seems to create a process pool instead of a thread pool. Hence the name of the argument passed here `self.num_threads` may not be the most accurate. Is that the case?

Reply:
yes it actually means num workers